dev/plots_api_strawman.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+
+# ======================================
+# Plots for Observation Planning Toolbox
+# ======================================
+
+# This is a strawman API for plotting with Astroplan.
+
+# Must first construct an Observer object using astropy.coordinates.
+
+from astroplan import Observer
+from astropy.coordinates import EarthLocation
+import astropy.units as u
+from pytz import timezone
+
+my_location = EarthLocation.from_geodetic(
```

Comment:
@eteq  Since I haven't messed around with `EarthLocation` a whole lot, I was going off of the example in the latest iteration to PR #4.  I'm not married to any of the syntax here, although :+1:  to `subaru_location = EarthLocation(lon='-155d28m48.900s', lat='+19d49m42.600s',height=4163*u.m)`.